app/src/main/res/layout/beer_layout.xml

```diff
@@ -22,15 +22,14 @@
             android:layout_height="wrap_content">
 
             <ImageView
-                android:id="@+id/imagePath"
-                android:layout_width="88dp"
-                android:layout_height="139dp"
-                android:layout_marginStart="34dp"
-                android:layout_marginTop="22dp"
+                android:id="@+id/imageIchnusa"
```

Comment:
Magari più che "imageIchnusa" che è molto specifico e non adatto ad altri tipi di birre, metterei qualcosa tipo "beerImage". Inoltre modificandolo solo qua e non in altre parti del codice dove viene usato (es. BeerAdapter), l'app non compila